tests/functional/manifest/expected/data/out/tables/testmetadata.manifest

```diff
@@ -1 +1 @@
-{"columns":["decimal","numeric","notnull","name"],"metadata":[{"key":"KBC.name","value":"testmetadata"}],"column_metadata":{"decimal":[{"key":"KBC.datatype.type","value":"numeric"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"10,2"}],"numeric":[{"key":"KBC.datatype.type","value":"numeric"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"18,0"}],"notnull":[{"key":"KBC.datatype.type","value":"character varying"},{"key":"KBC.datatype.nullable","value":false},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"200"}],"name":[{"key":"KBC.datatype.type","value":"character varying"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"200"}]}}
\ No newline at end of file
+{"columns":["notnull","name","decimal","numeric"],"metadata":[{"key":"KBC.name","value":"testmetadata"}],"column_metadata":{"notnull":[{"key":"KBC.datatype.type","value":"character varying"},{"key":"KBC.datatype.nullable","value":false},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"200"}],"name":[{"key":"KBC.datatype.type","value":"character varying"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"200"}],"decimal":[{"key":"KBC.datatype.type","value":"numeric"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"10,2"}],"numeric":[{"key":"KBC.datatype.type","value":"numeric"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"18,0"}]}}
```

Comment:
Ok, neviem ako je mozne, ze tam bolo predtym ine poradie, ale teraz to sedi s:
https://github.com/keboola/redshift-transformation/blob/26884cf178b3a4ffd8222b1bfce5ecde8a734af6/tests/functional/manifest/source/data/config.json#L29

Reply:
@ondrajodas sice vlastne to ani teraz nesedi, zle som to pozrel, ... ci sa mylim?

Reply:
no já právě koukám že to nesedí :D možná by bylo dobrý ještě sortnout ty sloupečky aby to nebylo pokaždé jinak

Reply:
no asi bz bolo idealne aby ich vratilo v takom poradi ako su definovane v DB, ... 

niekde tu pridar `ORDER BY`, treba poztiet aku to ma strukturu:
https://github.com/keboola/redshift-transformation/blob/26884cf178b3a4ffd8222b1bfce5ecde8a734af6/src/RedshiftTransformation.php#L165

Reply:
asi `order by ordinal_position;`

https://dataedo.com/kb/query/amazon-redshift/list-columns-names-in-specific-table

Reply:
jj to by asi taky šlo 👍